common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1534,6 +1534,24 @@
     ],
     "sqlState" : "42601"
   },
+  "IDENTITY_COLUMNS_DUPLICATED_DESCRIPTOR" : {
+    "message" : [
+      "Duplicated IDENTITY column descriptor: <descriptor>."
+    ],
+    "sqlState" : "42734"
```

Comment:
This state is quite specific to "column level" object names.
42601 should do here

Reply:
Changed to 42601.